app/src/main/java/org/schabi/newpipe/settings/ContentSettingsManager.kt

```diff
@@ -91,6 +91,10 @@ class ContentSettingsManager(private val fileLocator: NewPipeFileLocator) {
                         is String -> {
                             preferenceEditor.putString(key, value)
                         }
+                        is HashSet<*> -> {
+                            @Suppress("UNCHECKED_CAST")
```

Comment:
```suggestion
                            // There are currently only Sets with type String possible
                            @Suppress("UNCHECKED_CAST")
```

Adding a comment here why the warning is ignored